components/hubspot/sources/new-deal-in-stage/new-deal-in-stage.mjs

```diff
@@ -25,6 +25,7 @@ export default {
           pipeline: c.pipeline,
         }),
       ],
+      withLabel: false,
```

Comment:
I guess you can remove this property from here and if not needed also remove it from `propDefinitions` since this is the only place it's being used